moveos/moveos-server/protos/server.proto

```diff
@@ -0,0 +1,438 @@
+
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+package moveoss;
+
+import "google/protobuf/timestamp.proto";
+
+message Transaction {
```

Comment:
Is decided to use protobuf for serialization and deserialization?

Reply:
yes，the first version of server and cli uses grpc, so choose proto, and later versions can be updated as needed